<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoDialog</class>
 <widget class="QDialog" name="DemoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>上位机通信框架示例</string>
  </property>
  <widget class="QPushButton" name="pushButtonConnectDevice">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ConnectDevice</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>SetValue command example</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Value:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSetValue">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>234</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetValue">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>GetValue command example</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tempeture:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemperatureValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGettemperature">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>GetTempeture</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>ReportValue command example</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Report Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelReportValue">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelConnectStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
